RUNNING_INFERSHARP_ON_WINDOWS.md

```diff
@@ -0,0 +1,38 @@
+# How to Run Infer# locally on Windows via WSL2 (Windows Subsystem for Linux)
+
+## Prerequisite
+- [Enable WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+
+## Setup
+1. Download the Infer# binaries from [Infer# - v1.1](TBA).
+2. From Windows Command Prompt or PowerShell, enter: ```wsl.exe``` to open your default Linux distribution.
+3. Copy the _infersharp_ folder to _/home/_ and create a symlink for future use:
```

Comment:
Actually Windows user's directory can be accessed at path `/mnt/c/Users/username/`